source/Nucleus.lagda

```diff
@@ -0,0 +1,156 @@
+Ayberk Tosun, 15 October 2021
+
+Based on `ayberkt/formal-topology-in-UF`.
+
+\begin{code}[hide]
+
+{-# OPTIONS --without-K --exact-split --safe #-}
+
+open import SpartanMLTT
+open import UF-Base
+open import UF-PropTrunc
+open import UF-FunExt
+open import UF-PropTrunc
+
+module Nucleus
+        (pt : propositional-truncations-exist)
+        (fe : Fun-Ext)
+       where
+
+open import UF-Subsingletons
+open import UF-Subsingleton-Combinators
+open import UF-Subsingletons-FunExt
+
+open import Frame pt fe
+
+open AllCombinators pt fe
+
+\end{code}
+
+\begin{code}
+
+is-inflationary : (L : frame 𝓤 𝓥 𝓦) → (⟨ L ⟩ → ⟨ L ⟩) → Ω (𝓤 ⊔ 𝓥)
+is-inflationary L j = Ɐ x ∶ ⟨ L ⟩ , x ≤[ poset-of L ] j x
+
+is-idempotent : (L : frame 𝓤 𝓥 𝓦) → (⟨ L ⟩ → ⟨ L ⟩) → Ω (𝓤 ⊔ 𝓥)
+is-idempotent L j = Ɐ x ∶ ⟨ L ⟩ , j (j x) ≤[ poset-of L ] j x
+
+is-nuclear : (L : frame 𝓤 𝓥 𝓦) → (⟨ L ⟩ → ⟨ L ⟩) → Ω (𝓤 ⊔ 𝓥)
```

Comment:
Rename "is-nuclear" to "is-nucleus".

Reply:
Shouldn't "preserves-meets" be called "preserves-finite-meets"? Or "preserves-binary-meets"?